src/app/proposals/shared/utils.ts

```diff
@@ -153,3 +155,57 @@ export const isChecksumValid = (value: string, chainId?: string) => {
     checksumAddress(value as Address, chainId ? Number(chainId) : undefined) === value
   )
 }
+
+type ProposalSource = 'DAO' | 'TALLY' | 'UNKNOWN'
+interface ParsedDescription {
+  name: string
+  description: string
+  source: ProposalSource
+}
+
+export const parseProposalDescription = (description: string): ParsedDescription => {
+  // Constants
+  const MAX_NAME_LENGTH = 100
+  const TALLY_DOUBLE_SPACE = '  ' // Tally uses double spaces to separate name and description
```

Comment:
Rename the constant to something more meaningful like TALLY_DESCRIPTION_SEPARATOR or likewiise
Keep the comment 👍